students/thomasvad/lesson2/series.py

```diff
@@ -0,0 +1,73 @@
+def fibonacci(n):
+    """ compute the nth Fibonacci number """
+    l=[0,1]
```

Comment:
Spaces needed around assignment operator, as well as after comma.  Here and below